### PR TITLE
BLK-306, version updates

### PR DESCRIPTION
**Pull request by f1401martin**
looks like these changes are in order, @mattdodge I believe the repo can use a `1.1.0` tag as well, can you confirm?

### CODE REVIEW

**Comment by mattdodge** [comment]
Yeah, this is good to do. I think there is a Jenkins job which takes care of some of this stuff but I'm not 100% sure on that. @tyoungNIO will probably know more about how that works.

**Comment by f1401martin** [comment]
just saw `block_release_script` which automates this, so these may be redundant, @tlugger do you know anything about this?

**Comment by tlugger** [comment]
Yes, if I remember correctly, you want to PR everything except for version updates into master. Then you run a release on the `nio Blocks/_block_release` job for each of the 4 blocks in here specifying the new version. So I think you need to remove everything except for the property updates in this PR and that job will handle the rest.

File: local_publisher.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ class LocalPublisher(PubSubConnectivity, TerminatorBlock):
 
     Unlike the regular Publisher block, the one does not need data to be json
     """
-    version = VersionProperty("0.1.2")
+    version = VersionProperty("1.1.0")
```
Comment (tlugger) [inline local_publisher.py]: Change back to `0.1.2`

File: local_subscriber.py [hunk 2]
```diff
@@ -15,7 +15,7 @@ class LocalSubscriber(PubSubConnectivity, GeneratorBlock):
 
     Unlike the regular Subscriber block, the one does not need data to be json
     """
-    version = VersionProperty("0.1.2")
+    version = VersionProperty("1.1.0")
```
Comment (tlugger) [inline local_subscriber.py]: Change back to `0.1.2`

File: publisher.py [hunk 3]
```diff
@@ -15,7 +15,7 @@ class Publisher(PubSubConnectivity, TerminatorBlock):
         topic (str): Defines topic to use to publish signals.
 
     """
-    version = VersionProperty("1.0.2")
+    version = VersionProperty("1.1.0")
```
Comment (tlugger) [inline publisher.py]: Change back to `1.0.2`

File: release.json [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
   "nio/Publisher": {
     "language": "Python",
-    "version": "1.0.2",
+    "version": "1.1.0",
     "url": "git://github.com/nio-blocks/communication.git"
   },
   "nio/Subscriber": {
     "language": "Python",
-    "version": "1.0.2",
+    "version": "1.1.0",
```
Comment (tlugger) [inline release.json]: Revert all of these to their previous version numbers

File: spec.json [hunk 5]
```diff
@@ -1,19 +1,34 @@
 {
   "nio/LocalPublisher": {
-    "version": "0.1.2",
+    "version": "1.1.0",
```
Comment (tlugger) [inline spec.json]: Change back to `0.1.2`

File: spec.json [hunk 6]
```diff
@@ -28,20 +43,35 @@
     "commands": {}
   },
   "nio/LocalSubscriber": {
-    "version": "0.1.2",
+    "version": "1.1.0",
```
Comment (tlugger) [inline spec.json]: Change back to `0.1.2`

File: spec.json [hunk 7]
```diff
@@ -56,14 +86,27 @@
     "commands": {}
   },
   "nio/Publisher": {
-    "version": "1.0.2",
+    "version": "1.1.0",
```
Comment (tlugger) [inline spec.json]: Change back to `1.0.2`

File: spec.json [hunk 8]
```diff
@@ -78,14 +121,27 @@
     "commands": {}
   },
   "nio/Subscriber": {
-    "version": "1.0.2",
+    "version": "1.1.0",
```
Comment (tlugger) [inline spec.json]: Change back to `1.0.2`

File: subscriber.py [hunk 9]
```diff
@@ -14,7 +14,7 @@ class Subscriber(PubSubConnectivity, GeneratorBlock):
         topic (str): Defines topic to subscribe to in order to receive signals.
 
     """
-    version = VersionProperty("1.0.2")
+    version = VersionProperty("1.1.0")
```
Comment (tlugger) [inline subscriber.py]: Change back to `1.0.2`

**Pull request merged**